fbw-a380x/src/systems/instruments/src/ANC/instrument.tsx

```diff
@@ -0,0 +1,141 @@
+import { EventBus, FSComponent, HEventPublisher } from '@microsoft/msfs-sdk';
+
+var flightphase = {
+  "WelcomeOnboard": 0,
+  "SafetyBriefing": 0,
+  "CrewSeatsTakeoff": 0,
+  "SeatbeltSignOff": 0,
+  "StartingDecent": 0,
+  "CrewPrepareLanding": 0,
+  "ArrivalDeboarding": 0
+};
+/**
+ * Resetting the flightphase to simulate turnaround or new flight
+ */
+
+
+
+
+/**
+ * 
+ * @param varCat - Category if either L Var or A var 
+ * @param variableName - Name of the SimVar to be executed
+ * @param type - Type of SimVar
+ * @param value - Value of the SimVar, for L Vars consider only Numbers
+ */
+
+
+function setSimvarTrigger(varCat, variableName, type, value) {
+  if (varCat === "L") {
+    SimVar.SetSimVarValue(`L:${variableName}`, `${type}`, value);
+  } else if (varCat === "A") {
+    SimVar.SetSimVarValue(`${variableName}`, `${type}`, value);
+  }
+  setTimeout(function() {
+    SimVar.SetSimVarValue(`L:${variableName}`, `${type}`, 0);
+  }, 1e4);
+}
+class A380X_ANC extends BaseInstrument {
+
+  private readonly hEventPublisher: HEventPublisher;
+
+  constructor() {
+    super();
+  this.bus = new EventBus();
+  
+  this.airspeedPublisher = new SimVarPublisher(new Map([
+      ["indicated_airspeed", { name: "AIRSPEED INDICATED", type: "knots" }],
+      ["true_airspeed", { name: "AIRSPEED TRUE", type: "knots" }],
+      ["mach", { name: "AIRSPEED MACH", type: "number" }],
+      ["pressure_altitude", { name: "PRESSURE ALTITUDE", type: "feet" }],
+      ["vertical_speed", { name: "VERTICAL SPEED", type: "feet per minute" }],
+      ["total_air_temperature", { name: "TOTAL AIR TEMPERATURE", type: "celsius" }],
+      ["static_air_temperature", { name: "AMBIENT TEMPERATURE", type: "celsius" }]
+    ]), this.bus);
+
+    this.onGroundPublisher = new SimVarPublisher(new Map([
+      ["is_on_ground", { name: "SIM ON GROUND", type: "number" }]
+    ]), this.bus);
+
+    this.LVarPublisher = new SimVarPublisher( new Map([
+      ["seatbelt_sign_on", { name: "L:XMLVAR_SWITCH_OVHD_INTLT_SEATBELT_Position", type: "number" }],
+      ["passengers_boarding", { name: "L:A32NX_SOUND_PAX_BOARDING", type: "number" }],
+      ["passengers_deboarding", { name: "L:A32NX_SOUND_PAX_DEBOARDING", type: "number" }]
+    ]), this.bus);
+    this.isOnGround = 0;
+    this.indicatedAirspeed = 0;
+    this.pressureAltitude = 0;
+    this.verticalSpeed = 0;
+    this.leftParkingGate = 0;
+    this.seatbeltsignState = 0;
+    this.passengerboarding = 0;
+    this.passengerdeboarding = 0;
+  }
+
+  get templateID(): string {
+    return 'A380X_ANC';
+  }
+  
+ public connectedCallback(): void {
+    super.connectedCallback();
+    this.airspeedPublisher.startPublish();
+    this.onGroundPublisher.startPublish();
+    this.LVarPublisher.startPublish();
```

Comment:
Use an InstrumentBackplane (add to the one in extras host when this is moved there).